lib/datadog/lambda.rb

```diff
@@ -112,7 +113,10 @@ def self.gen_enhanced_tags(context)
         memorysize: context.memory_limit_in_mb,
         cold_start: @is_cold_start,
         runtime: "Ruby #{RUBY_VERSION}",
-        resource: context.function_name
+        resource: context.function_name,
+        dd_trace: Gem.loaded_specs['ddtrace'].version,
```

Comment:
ddtrace might not be installed beside datadog-lambda-rb, (we test to see if it's available before using it). It might make sense to handle the case where it isn't present.

Reply:
added!